op-chain-ops/genesis/config.go

```diff
@@ -233,6 +233,18 @@ type DeployConfig struct {
 	// RequiredProtocolVersion indicates the protocol version that
 	// nodes are recommended to adopt, to stay in sync with the network.
 	RecommendedProtocolVersion params.ProtocolVersion `json:"recommendedProtocolVersion"`
+	// ProofMaturityDelaySeconds is the number of seconds that a proof must be
+	// mature before it can be used to finalize a withdrawal.
+	ProofMaturityDelaySeconds uint64 `json:"proofMaturityDelaySeconds"`
+	// DisputeGameFinalityDelaySeconds is an additional number of seconds a
+	// dispute game must wait before it can be used to finalize a withdrawal.
+	DisputeGameFinalityDelaySeconds uint64 `json:"disputeGameFinalityDelaySeconds"`
+	// RespectedGameType is the dispute game type that the OptimismPortal
+	// contract will respect for finalizing withdrawals.
+	RespectedGameType uint32 `json:"respectedGameType"`
+	// UseOptimismPortal2 configures whether or not to use the OptimismPortal2
+	// contract when deploying the system.
+	UseOptimismPortal2 bool `json:"useOptimismPortal2"`
 
 	// When Cancun activates. Relative to L1 genesis.
```

Comment:
nit: no update to `Check` function means that there is no assertions in our deploy config json tests

Reply:
Added checks